flags.css

```diff
@@ -1,5 +1,17 @@
 /* Indonesia */
-
+#indonesia {
+    height: 200px;
```

Comment:
in the instructions, it does say to use pixels.  however vmin might be better for real usage.

Also, all your flags are going to be the same size, for the most part.  So you could add the size to the .flagContainer and get more bang for your buck.